thaomonster/ACNH-Collections
# Feature/cypress testing

### What does this PR do? 
>Clearly and concisely describe the feature. 

This PR adds E2E testing with cypress.

### Is this a feature or Fix/Refactor?  
Feature

### Problems Encountered & Resulting Solutions  
>State any problems encountered 
>Describe your code changes in detail for reviewers.

Having difficulty stubbing.

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?
`cypress/fixtures/villagerData.json `
`cypress/integration/profilePage-test.js `
`cypress/integration/villagerList-test.js`
beginning to end

### How should this be tested?
>What steps need to be taken to confirm functionality? 
1. `git fetch` branch
2. run `npm i`
3. run `nom run cypress`

### Steps To Take
>Any questions and possible steps moving forward.

Try to get stub working.

### This PR is related to which issues:
>Tag issue here: `Closes #<issue_number>`
Closes #18
